TypeCobol.LanguageServer/WorkspaceProject.cs

```diff
@@ -174,10 +174,13 @@ public void Configure(DocumentFormat format, TypeCobolOptions options, AnalyzerP
             var oldCopyFolders = GetCopyFolders();
             bool updateCopyFolders = false;
             var newCopyFolders = oldCopyFolders.ToList();
-            if (copyFolders != null && AreCopyFoldersDifferent(oldCopyFolders, copyFolders))
+            if (copyFolders != null)
             {
-                updateCopyFolders = true;
-                newCopyFolders = copyFolders;
+                newCopyFolders = copyFolders.Union(new []{ Workspace.GetDefaultCopyFolder }, StringComparer.OrdinalIgnoreCase).ToList();
```

Comment:
You have to use a local variable  before assigning `newCopyFolders` because the value is used even when the boolean `updateCopyFolders` is false.
Also I use InvariantCultureIgnoreCase for paths and OrdinalIgnoreCase for source code names but I'm not sure it is required.

Reply:
if `AreCopyFoldersDifferent(oldCopyFolders, newCopyFolders)` is false, then we can keep the value of newCopyFolders because it's the same as `oldCopyFolders.ToList()`.
Done for InvariantCultureIgnoreCase : ac4aca9eee24c42ad79ca7c90ff65fc5ddc8059a.